source/level_set_okz_advance_concentration.cc

```diff
@@ -355,8 +355,11 @@ struct AdvanceConcentrationMatrix
 // @sect4{LevelSetOKZSolverAdvanceConcentration::advance_concentration}
 template <int dim>
 void
-LevelSetOKZSolverAdvanceConcentration<dim>::advance_concentration()
+LevelSetOKZSolverAdvanceConcentration<dim>::advance_concentration(const double dt)
 {
+  this->time_stepping.set_time_step(dt);
+  this->time_stepping.next();
```

Comment:
FYI @mschreter Now this branch works again as it used to. Feel free to push again!